code/Attack.ipynb

```diff
@@ -0,0 +1,1620 @@
+{
```

Comment:
<p>Give a run down of what we're downloading here. Also, the kind of models being evaluated here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/robustness-foundation-models/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>